### DISCUSSION TITLE
library/EEPROM

### DISCUSSION THREAD

**Discussion by giscus** [py32duino/document#7] [comment]
# library/EEPROM

简介 EEPROM 是一种存储器，其值在电路板断电时保留。在 PY32Duino 中，没有内置的 EEPROM，但是可以使用 Flash 模拟 EEPROM。一般来说，我们采用内置 flash 的最后一个 page 扇区（或者是其它可擦写的最小单位）来模拟。 API 要使用此库，请在代码顶部包含头文件： read() 从 EEPROM 读取一个字节。 ...

https://arduino.py32.org/library/EEPROM.html

**Comment by yong-pii** [comment]
Get!
